README.md

```diff
@@ -1018,7 +1018,8 @@ interface FCPAttribution {
   fcpEntry?: PerformancePaintTiming;
   /**
    * The `navigation` entry of the current page, which is useful for diagnosing
-   * general page load issues.
+   * general page load issues. This can be used to access `serverTiming` for example:
+   * navigationEntry.serverTiming
```

Comment:
A matter of taste. The [OptionalChaining `?` operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) is only needed if `navigationEntry` is undefined. But I think in terms of docs here it's fine to exclude it.

It could also we used like this without the `?`, for example:

```js
const myServerTiming = navigationEntry ? navigationEntry.serverTiming : '(not set)'
```

Reply:
Phil used it in his example code so probably worth sticking with his preference 😛